configuration-sample/ods-core.env.sample

```diff
@@ -54,8 +54,8 @@ SONAR_DATABASE_NAME=sonarqube
 SONAR_DATABASE_PASSWORD_B64=changeme
 SONAR_DATABASE_USER=sonarqube
 
-SONAR_DISTRIBUTION_URL=https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-7.3.zip
-SONAR_VERSION=7.3
+SONAR_DISTRIBUTION_URL=https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-7.6.zip
```

Comment:
Please make this 7.9. This is the latest LTS release and tested by us.

Reply:
Changed to 7.9